canister/src/block_header_store.rs

```diff
@@ -41,6 +41,16 @@ impl BlockHeaderStore {
         }
     }
 
+    pub fn insert2(&mut self, block_hash: BlockHash, header_bytes: Vec<u8>, height: Height) {
```

Comment:
Oops, I forgot to rename this method - thanks for catching that. I think it would be confusing to guard it by the `file_memory` feature, since its functionality is unrelated to a `FileMemory`.